timemachine/fe/topology.py

```diff
@@ -65,6 +67,21 @@ def __init__(self, host_potentials, guest_topology):
     def get_num_atoms(self):
         return self.num_host_atoms + self.guest_topology.get_num_atoms()
 
+    def get_component_idxs(self) -> List[NDArray]:
+        """
+        Return the atom indexes for each component of
```

Comment:
nit: indexes -> indices

Reply:
How are the "components" here defined? Later on we may want to define water component, protein component, mol_a, mol_b component, etc.? Right now host = water+protein

Reply:
> How are the "components" here defined?

Currently environment + ligands.

> Later on we may want to define water component, protein component, mol_a, mol_b component, etc.? Right now host = water+protein

This is true. It could return a dataclass instead to make it more explicit what is represented and easier to add more components in the future.